### ISSUE TITLE
Unsupported endpoint issue

### ISSUE THREAD

**Issue by noidsign** [geduldig/TwitterAPI#104]
Hi, I'm a super newbie in programming.
I'm working on a research project where I'm trying to get all the tweets with specifics hashtags or keywords. I got myself a Premium Twitter and was trying to use your TwitterAPI to access historical tweets. 
From Stackoverflow I found [this](https://stackoverflow.com/questions/50272868/using-tweepy-to-access-twitters-premium-api) and it was extremely helpful, but I still couldn't retrieve the tweets because I got the following error:
```
File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/TwitterAPI/TwitterAPI.py", line 111, in request
raise Exception('Endpoint "%s" unsupported' % endpoint)
```
I've tried both just putting the label of my environment and `label.json` as endpoints, but it doesn't work either way. 

I'm not sure what I'm doing wrong, and I'd love if you could help me.

I have further questions on how to retrieve tweets only between certain dates, but that's second order compared to why I get the endpoint error.

Thank you.

**Comment by hshahsahebi** [comment]
Why this case is closed? Did you find any workaround?

**Comment by geduldig** [comment]
A workaround for what exactly?

**Comment by hshahsahebi** [comment]
I was able to fix the issues I faced during integrating your library which is working fine now. However, premium/sandbox api need more clarification.
For instance, to add the label for the sandbox/premium API we need to add a colon before the label name. The following string is the correct API endpoint for premium access which I figured it out by reviewing the TwitterAPI class:
`tweets/search/30day/:environment`

Thank you
